assembly/src/assembler/instruction/crypto_ops.rs

```diff
@@ -191,3 +178,22 @@ fn read_mtree_node(span: &mut SpanBuilder) {
     // MRUPDATE: [V_old, d, i, R, V_new, ...]
     span.push_op_many(Read, 4);
 }
+
+/// Update a node in the merkle tree. The `copy` flag will be passed as argument of the `MrUpdate`
+/// operation.
+fn update_mtree(span: &mut SpanBuilder, copy: bool) {
+    #[rustfmt::skip]
+    span.push_ops([
```

Comment:
question: why do we sometimes put operations into a separate array (e.g., `let ops = [...];`) but here we pass it directly to `push_ops([...])`?

Reply:
I would rather not have this separated, but we currently have a problem in Rust parser.

If the expression is returning in a function, then we can't enable attributes - without activating the `stmt_expr_attributes` unstable feature of the language

https://github.com/rust-lang/rust/issues/15701

Reply:
Makes sense!